5001-6000/LIT5849GenealogyApostles.xml

```diff
@@ -1,77 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
```

Comment:
https://betamasaheft.eu/Guidelines/?q=delete&id=create
If it is clear that a record is superfluous, delete it, properly document this in the commit, make sure the deleted identifier is not used elsewhere, update existing references, but also **add a relation in the record which survives**, like the following.


                  `<relation name="betmas:formerlyAlsoListedAs" active="IDofthesurvivedrecord" passive="IDofthedeletedrecord"></relation>`

If you are sure that it is a doublet and delete a record make sure to update all records pointing to it (BNFet64, BAVet10, LIT2422Tarika)!

Reply:
ok